marl_eval/json_tools/json_logger.py

```diff
@@ -19,19 +19,19 @@
 from typing import Dict, Optional
 
 
-class JsonWriter:
-    """Writer to create json files for reporting experiment results.
+class JsonLogger:
+    """Logger to create JSON files for reporting experiment results.
 
-    This writer follow suggested marl-eval protocol and was adapted from
-    the implementation found in BenchMARL which can be viewed
-    at https://tinyurl.com/2t6fy548
+    This logger follows the suggested marl-eval protocol and was adapted
+    from the implementation found in BenchMARL which can be viewed at:
+    https://tinyurl.com/2t6fy548
 
     Args:
-        path (str): where to write the file
-        algorithm_name (str): algorithm name
-        task_name (str): task name
-        environment_name (str): environment name
-        seed (int): random seed of the experiment
+        path (str): folder path for where to save metrics.json.
```

Comment:
Sorry my mistake, please change it again :smile: 

```suggestion
        path (str): folder path for saving the `metrics.json` file.
```